devices/datek.js

```diff
@@ -243,4 +243,19 @@ module.exports = [
         },
         exposes: [e.contact(), e.battery_low(), e.tamper(), e.temperature()],
     },
+    {
+        zigbeeModel: ['Contact Switch'],
+        model: 'Contact Switch',
```

Comment:
Can you update this to the model listed on the site/device/box? Also update the picture in https://github.com/Koenkk/zigbee2mqtt.io/pull/2043/files after that.

Reply:
This is in fact the model name of the device :) Do you want me to change the name of the picture to DATEK_CONTACT_SWITCH maybe?

![Screenshot 2023-05-20 at 18 42 15](https://github.com/Koenkk/zigbee-herdsman-converters/assets/44670294/e0007d60-3657-4c20-b098-3306d49c7f00)

Reply:
Just found the name you are after on the box :) HSE2936T
Changed it in commit [7dc7589](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5756/commits/7dc7589636e84baed22f19fde11a919a19b3c305)

Reply:
![abbc9a7e9c984252e90d06aac74bca662930debced678eedb9544306bdce6ce1](https://github.com/Koenkk/zigbee-herdsman-converters/assets/44670294/9ef6ad9e-3d49-44be-9f79-330cdff51297)